core/src/main/scala/cats/data/RWST.scala

```diff
@@ -0,0 +1,550 @@
+package cats
+package data
+
+import cats.functor.{ Contravariant, Bifunctor, Profunctor }
+import cats.syntax.either._
+
+/**
+  * Represents a stateful computation in a context `F[_]`, over state `S`, with an initial environment `E`, an accumulated log `L` and a result `A`.
```

Comment:
Minor thing, but I believe in cats the convention is javadoc-style doc comments, rather than scaladoc-style.  Thusly:

```scala
/**
 * Docs here
 */
```

Rather than

```scala
/**
  * Boo
  */
```